pkg/serviceaccount/jwt.go

```diff
@@ -287,6 +289,8 @@ func (j *jwtTokenAuthenticator) AuthenticateToken(ctx context.Context, tokenData
 	tokenAudiences := authenticator.Audiences(public.Audience)
 	if len(tokenAudiences) == 0 {
 		// only apiserver audiences are allowed for legacy tokens
+		klog.Warningf("Legacy service account token used, service account: %s", public.Subject)
```

Comment:
This is going to be *extremely* noisy in most installations... I would lower the verbosity significantly. Also, add a TODO to add an audit annotation once https://github.com/kubernetes/kubernetes/pull/89305 is complete

Reply:
Switched to Info. Also applied to the log for stale token.